src/sections/Meshery/Meshery-integrations/Individual-Integrations/Component.style.js

```diff
@@ -0,0 +1,55 @@
+import styled from "styled-components";
+
+export const ComponentsWrapper = styled.div`
+  .heading {
+    text-align: center;
+    margin-top: 7rem;
+
+    h1 {
+    }
+
+    h2 {
+      font-weight: normal;
+    }
+  }
+
+  .componentimg {
+    width: 80px;
+    height: 80px;
+  }
+  .componentimg img {
+    width: 100%;
+    height: 100%;
+    object-fit: contain;
+  }
+
+  .componentsSection {
+    display: flex;
+    flex-wrap: wrap;
+    gap: 1.5rem;
+    padding: 3rem 2rem 5rem 2rem;
+    justify-content: center;
+  }
+  .maincontainer {
+    display: flex;
+    align-items: center;
+    gap: 1rem;
+    background-color: ${props => props.theme.grey212121ToGreyEEEEEE};
+    padding: 0.5rem 1rem;
+    border-radius: 0.25rem;
```

Comment:
```suggestion
    border-radius: 0.85rem;
```

Reply:
Just a guess that this size might fit a bit better...

Reply:
commited.